measure/endurance-snapshot

```diff
@@ -475,8 +467,14 @@ fi
 echo "- Disk usage"
 LC_ALL=C df -k -P > $path/df
 
-echo "- statefs"
-dump_statefs_value namespaces/Screen/Blanked
+echo "- Blanked state"
+blanked_state=$(qdbus --system com.nokia.mce /com/nokia/mce/request com.nokia.mce.request.get_display_status)
```

Comment:
Is qdbus installed by default / pulled in by endurance packages?
Is it really in default path?

[I think I've always installed qdbus manually and added symlink to /usr/lib/qt/bin/qdbus from something in path... probably not how it should be done]

Reply:
Relying fully on mce means no need for qdbus, so hopefully this point becomes moot.